XMonad/Actions/LinkWorkspaces.hs

```diff
@@ -14,7 +14,7 @@
 --
 -----------------------------------------------------------------------------
 
-{-# LANGUAGE DeriveDataTypeable  #-}
+
```

Comment:
I also think these (when the pragma is declared after the module header) weren't picked up